test/unit/spec/v1/PrimaryAuth_spec.js

```diff
@@ -1495,6 +1495,12 @@ Expect.describe('PrimaryAuth', function() {
           });
       });
       itp('shows beacon-loading animation when primaryAuth is submitted (with deviceFingerprint)', function() {
+        spyOn(DeviceFingerprint, 'generateDeviceFingerprint').and.callFake(function() {
```

Comment:
Adding missing mocks.
Without this fix, test tries to run real fingerprint for 2s:
https://github.com/okta/okta-signin-widget/blob/bdf57aa226eeeb4ab5724ad1113e5d63d35d08e0/src/v1/util/DeviceFingerprint.js#L96-L99
But default wait timeout is also 2s
https://github.com/okta/okta-signin-widget/blob/bdf57aa226eeeb4ab5724ad1113e5d63d35d08e0/test/unit/helpers/util/Expect.js#L12

Reply:
Nice catch!